.github/workflows/reusable_gpu.yml

```diff
@@ -4,7 +4,25 @@
 
 name: GPU
 
-on: [workflow_call]
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: Provider name
+        type: string
+        required: true
+      os:
+        description: A list of OSes
+        type: string
+        required: true
+      build_type:
```

Comment:
I find this input unused by other workflows. Perhaps I've omitted some usage?

Reply:
you're right - at this moment it's used only its default value; I was using this to skip some troublesome builds, like exclude Debug from L0, which I found useful and left here for others to use, if needed